gp/cgp_node.py

```diff
@@ -188,7 +188,7 @@ def format_output_str(self, graph):
         self._output_str = '{}'.format(self._output)
 
     def format_output_str_torch(self, graph):
-        self._output_str = 'self._p{}'.format(self._idx)
+        self._output_str = 'torch.cat(x.shape[0] * [self._p{}])'.format(self._idx)
```

Comment:
i don't like this use of `x`, it's not apparent here where it comes from. isn't it enough to have as output value 2d tensor that torch can resize as necessary, i.e., `format_parameter_str` should be adjusted, rather than this function?

Reply:
I don't know how to do that in the way you're implying.
If the parameter is a 2d tensor, this doesn't work in combination with an operation that uses the input.

Also, x is used in CPGInputNode in a similar manner, such that it is not clear where x comes from.

Reply:
wow, this took me some time to process. i understand the problem now, but suggest a slightly simpler version (using `expand` instead of the list constructor)

Reply:
Yep, that's better, I committed the change.